### PR TITLE
Create game

### PR DESCRIPTION
**Pull request by elijahkim**

### CODE REVIEW

File: lib/game/game_manager.ex [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  end
+
+  def get_games do
+    GenServer.call(@name, {:get_games})
+  end
+
+  ##server callbacks
+
+  def init(:ok) do
+    state = %{games: %{}}
```
Comment (timlombardo) [inline lib/game/game_manager.ex:27]: It seems like the `games:` key isn't necessary here. The entire state map represents games so it seems odd to have an individual key for a single nested map.

Were you planning on adding another key to hold more state?

File: lib/game/game_manager.ex [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  def start_link do
+    GenServer.start_link(@name, :ok, name: @name)
+  end
+
+  def create_game do
+    Reactor.Random.random_string(10)
+    |> create_game
+  end
+
+  defp create_game(id) do
```
Comment (cjbell) [inline lib/game/game_manager.ex]: This method should probably be public?

**Comment by cjbell** [comment]
One nit but looks good!

File: lib/game/game_manager.ex (hunk 1 above)
Comment (elijahkim) [inline lib/game/game_manager.ex:27]: maybe. It does make it easier to add keys later

Comment (timlombardo) [inline lib/game/game_manager.ex:27]: yagni...

**Pull request merged**